tsconfig.json

```diff
@@ -22,9 +22,9 @@
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
     /* Modules */
-    "module": "ESNext", /* Specify what module code is generated. */
+    "module": "NodeNext", /* Specify what module code is generated. */
```

Comment:
Are you sure that this will work in a browser? Are there tests that check that the resolver is usable in (preferable different kinds of) browsers? This is often solved by having multiple `tsconfig.json` files and explicitly passing the name of the `tsconfig` you want to use to the `tsc` command.

Reply:
the test environment is configured as jsdom, but I agree some more thorough testing should be performed prior to release.